src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1331,6 +1331,179 @@ public async Task TestRuntimeBaseRouteInNextLinkForPaginatedRestResponse()
             }
         }
 
+        /// <summary>
+        /// Validates the Location header field returned as part of POST requests.
+        /// </summary>
+        /// <param name="entityType">Type of the entity</param>
+        /// <param name="requestPath">Request path for performing POST API requests on the entity</param>
+        [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        [DataRow(EntitySourceType.Table, "/api/Book", DisplayName = "Location Header validation - Table, Base Route not configured")]
+        [DataRow(EntitySourceType.StoredProcedure, "/api/GetBooks", DisplayName = "Location Header validation - Stored Procedures, Base Route not configured")]
+        public async Task ValidateLocationHeaderFieldForPostRequests(EntitySourceType entityType, string requestPath)
+        {
+
+            GraphQLRuntimeOptions graphqlOptions = new(Enabled: true);
```

Comment:
Why set graphql to enabled? can disable it and not have extra code defining an entity's graphQL metadata. That slightly distracts from the test because setting that metadata makes me think it is relevant in testing.

Reply:
Agreed. I've disabled graphql